### ISSUE TITLE
Reactive Time Out Not Work

### ISSUE THREAD

**Issue by khrisnaars** [moaazsidat/react-native-qrcode-scanner#220]
Im use reactive time out and not work when i set the time

below is my qrcodescanner attribute,and how i can reactive it manualy i try to set state on reactive but didnt work

        onRead={this.barcodeRecognized}
        flashMode={RNCamera.Constants.FlashMode.auto}
        reactivate={true}
        showMarker={true}
        reactivateTimeout={5}
        markerStyle={Styles.marker}
        checkAndroid6Permissions={checkAndroidPermission}

**Comment by moaazsidat** [comment]
Will accept PRs fixing this.

**Comment by acerbastimur** [comment]
still doesn't work..

**Comment by renatobenks** [comment]
the `reactivateTimeout` prop seems to be working fine in the latest version. so, we can you both describe a little better what is actually happening in the unexpected behaviour that you guys are facing on? and what's the version that you both are using?

**Comment by valn1** [comment]
yeah, it is not working as intended, like no matter how big is the timeout you set, it will reactivate right after scanning a code

<QRCodeScanner
                    reactivateTimeout={20000}
                    ref={(node) => { this.scanner = node }}>

**Comment by MarcelSchmidberger** [comment]
I had the same Issue, but with an upgrade to 1.5.3 the Problem was solved and the timeout is now working correctly.

```js
 <QRCodeScanner
  onRead={(e) => onScan(e)}
  reactivate={true}
  reactivateTimeout={2000}
  />
```
